recipes/mu-repo/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "mu_repo" %}
```

Comment:
Sure, no problem (this whole deal with `_` and `-` is confusing in this package I admit).

https://github.com/conda-forge/mu_repo-feedstock does not seem to have been rendered though. @jakirkham do you mean for me to open another PR?